kuma/static/js/contribution-handler.js

```diff
@@ -412,7 +447,7 @@
         // if not already initialised
         if (stripeHandler === null) {
             // initialise handler
-            stripeHandler = initStripeHandler();
+            // stripeHandler = initStripeHandler();
```

Comment:
@joshJarr From the diff it looks like this line is commented out. So, if `stripeHandler` is `null` nothing is going to happen. Is this intentional? If so, and this is handled somewhere else in the code, then it is probably best to completely remove this piece of code?

Reply:
Hi Schalk. Yep, I think this `if` statement is redundant and slipped through. `stripeHandler` will be set as part of `getStripeCheckoutScript()` when the popover is expanded. We then check if it's not `null` in `onSubmit` then the form is submitted.

I think, as a further improvement to this, we might consider showing an error message if the script fails to load for whatever reason or if it hasn't loaded by the time the user clicks submit, but I think this is an adjustment we can probably follow up with next week.